.github/workflows/scheduled_lints.yml

```diff
@@ -41,8 +41,8 @@ jobs:
       - name: Setup Node and cache
         uses: actions/setup-node@v4
         with:
-          node-version: 16
-          cache: 'yarn'
+          node-version: 18
```

Comment:
Why not LTS? Or even `v20`?

Reply:
Because the `hardhat@2.12.7` doesn't support 20.x.x